docs/06_how-to-guides/40_multi-index/how-to-modify-data-in-a-multi-index-table.md

```diff
@@ -31,11 +72,21 @@ To modify data in the multi index table defined in the above tutorial, you will
   check( itr != testtab.end(), "user does not exist in table" );
 
 +  testtab.modify( itr, _self, [&]( auto& row ) {
-+    row.secondary = user;
 +    row.datum = value;
 +  });
 }
 ```
 
+Now you have implemented a new action `mod`, which when sent to the blockchain will update the `datum` data member for user object identified by `user` name with the `value` sent as parameter.
```

Comment:
maybe

Now you have implemented a new action `mod`. Call `mod` to update the `datum` data member for THE user object identified by THE `user` name parameter.